### PR TITLE
stack: add enable + disable commands

### PR DESCRIPTION
**Pull request by lexton**

### CODE REVIEW

File: internal/cmd/stack/enable.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		return fmt.Errorf("expected zero arguments but got %d", nArgs)
+	}
+
+	var mutation stackDisableMutation
+	variables := map[string]interface{}{
+		"stack": graphql.ID(stackID),
+	}
+
+	return authenticated.Client.Mutate(cliCtx.Context, &mutation, variables)
+}
```
Comment (peterdeme) [inline internal/cmd/stack/enable.go:52]: How about this? A bit less code duplication.

```go
func enable(cliCtx *cli.Context) error {
	return enableDisable[stackEnableMutation](cliCtx)
}

func disable(cliCtx *cli.Context) error {
	return enableDisable[stackDisableMutation](cliCtx)
}

func enableDisable[T any](cliCtx *cli.Context) error {
	stackID := cliCtx.String(flagStackID.Name)

	if nArgs := cliCtx.NArg(); nArgs != 0 {
		return fmt.Errorf("expected zero arguments but got %d", nArgs)
	}

	var mutation T
	variables := map[string]interface{}{
		"stack": graphql.ID(stackID),
	}

	return authenticated.Client.Mutate(cliCtx.Context, &mutation, variables)
}
```

**Comment by peterdeme** [comment]
@lexton thanks for the contribution 🙏

File: internal/cmd/stack/enable.go (hunk 1 above)
Comment (lexton) [reply internal/cmd/stack/enable.go:52]: Sounds good - I'm not fussed, easy enough to keep clean. I copied the lock.go code to implement this.

**Comment by peterdeme** [comment]
Released in [v0.16.0](https://github.com/spacelift-io/spacectl/releases/tag/v0.16.0)

**Pull request merged**